libr/core/rtr_http.c

```diff
@@ -81,6 +81,7 @@ static int r_core_rtr_http_run(RCore *core, int launch, int browse, const char *
 		}
 		memset (&so, 0, sizeof (so));
 	}
+	s->proto = R_SOCKET_PROTO_TCP;
```

Comment:
maybe r_socket_new should set TCP as default protocol. because protocl=0 means nothing